core/metrics-core-service/src/test/java/org/hawkular/metrics/core/service/metrics/GaugeITest.java

```diff
@@ -346,68 +346,128 @@ public void addAndFetchGaugeDataAggregates() throws Exception {
         assertMetricIndexMatches(tenantId, GAUGE, singletonList(new Metric<>(m1.getMetricId(), m1.getDataPoints(), 7)));
     }
 
-    @Test
-    public void addAndCompressData() throws Exception {
-        DateTime dt = new DateTime(2016, 9, 2, 14, 15); // Causes writes to go to compressed and one uncompressed row
-        DateTimeUtils.setCurrentMillisFixed(dt.getMillis());
-
-        DateTime start = dt.minusMinutes(30);
-        DateTime end = start.plusMinutes(20);
-
-        MetricId<Double> mId = new MetricId<>(tenantId, GAUGE, "m1");
-
-        metricsService.createTenant(new Tenant(tenantId), false).toBlocking().lastOrDefault(null);
-
-        Metric<Double> m1 = new Metric<>(mId, asList(
-                new DataPoint<>(start.getMillis(), 1.1),
-                new DataPoint<>(start.plusMinutes(2).getMillis(), 2.2),
-                new DataPoint<>(start.plusMinutes(4).getMillis(), 3.3),
-                new DataPoint<>(end.getMillis(), 4.4)));
-
-        Observable<Void> insertObservable = metricsService.addDataPoints(GAUGE, Observable.just(m1));
-        insertObservable.toBlocking().lastOrDefault(null);
-
-        DateTime startSlice = DateTimeService.getTimeSlice(start, CompressData.DEFAULT_BLOCK_SIZE);
-        DateTime endSlice = startSlice.plus(CompressData.DEFAULT_BLOCK_SIZE);
+//    @Test
```

Comment:
Looks like this test needs to be updated. When I uncommented and ran, it fails with:

```
java.lang.AssertionError: Not completed! (0 completions) (+1 error)
	at org.hawkular.metrics.core.service.metrics.GaugeITest.addAndCompressData(GaugeITest.java:397)
Caused by: java.util.concurrent.ExecutionException: com.datastax.driver.core.exceptions.InvalidQueryException: unconfigured table data_temp_2017063020
Caused by: com.datastax.driver.core.exceptions.InvalidQueryException: unconfigured table data_temp_2017063020
```